packages/kbn-apm-synthtrace/src/config/good.yaml

```
@@ -0,0 +1,9 @@
+schedule:
+  - scenario: "good"
+    start: "now-5m"
+    end: "now-3m"
+    randomness: 0.2
+    eventsPerCycle:
+      start: 50
+      end: 2
+      method: "exp"
```

If I read this right, this is another way of specifying throughput. If that's the case I suggest calling it `rate` with the option of specifying `interval` or  alternatively specifying `ratePerMin`:

```suggestion
    ratePerMin:
      start: 50
      end: 2
      method: "exp"
```